<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="4,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,2">
          <item>
           <widget class="QDateEdit" name="dateEdit_cur"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_cur">
            <property name="title">
             <string>实时收益曲线</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_index">
            <property name="title">
             <string>叠加指数</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>30</y>
               <width>568</width>
               <height>18</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="checkBox_index_sh">
                <property name="text">
                 <string>上证综指(000001.SH)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_index_sz">
                <property name="text">
                 <string>深证成指(399001.SZ)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_index_zx">
                <property name="text">
                 <string>中小板指(399005.SZ)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_index_cy">
                <property name="text">
                 <string>创业板指(399006.SZ)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_cur">
          <column>
           <property name="text">
            <string>代码</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>买入日</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>卖出日</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>利润(%)</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
          <item>
           <widget class="QDateEdit" name="dateEdit_daily"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_daily">
            <property name="title">
             <string>某日收益曲线</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QDateEdit" name="dateEdit_oper"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>买入股票：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_buy"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>卖出股票:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_sell"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_oper">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1282</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
